.bazelrc

```diff
@@ -20,4 +20,65 @@ test --flaky_test_attempts=3
 
 # The unified launcher runs in Python 2 host configuration
 # https://github.com/bazelbuild/bazel/issues/7899
-build --host_force_python=PY2 
\ No newline at end of file
+build --host_force_python=PY2 
+
+# ------------------------
+# General RBE configuration
+# ------------------------
+build:remote --jobs=50
+
+# Java toolchain setup
+build:remote --java_toolchain=@bazel_tools//tools/jdk:toolchain_hostjdk8
+build:remote --host_java_toolchain=@bazel_tools//tools/jdk:toolchain_hostjdk8
+build:remote --host_javabase=@rbe_default//java:jdk
+build:remote --javabase=@rbe_default//java:jdk
+
+# C++ toolchain setup
+build:remote --extra_toolchains=@rbe_default//config:cc-toolchain
+build:remote --crosstool_top=@bazel_toolchains//configs/ubuntu16_04_clang/9.0.0/bazel_1.0.0/cc:toolchain
```

Comment:
Seems like the latest one for Bazel 2.1.0 is:

```
@bazel_toolchains//configs/ubuntu16_04_clang/10.0.0/bazel_2.1.0/cc:toolchain
```

Reply:
Done, using the default toolchain alias. 

I had to update the base docker image that the android_test image was built from, so that it contains clang 10.